src/styles/SearchForm.css

```diff
@@ -0,0 +1,40 @@
+.search-form {
+  margin-top: 2rem;
+}
+
+.search-form__input {
+  box-sizing: border-box;
+  border: 2px solid #ddd;
+  border-radius: 99px;
+  padding: 1rem 1.5rem;
+  background-color: #fff;
+  width: 100%;
+}
+
+.search-form__button {
+  border: 2px solid #153253;
+  border-radius: 99px;
+  padding: 1rem 1.5rem;
+  font-weight: 700;
+  color: #fff;
+  background-color: #153253;
+  cursor: pointer;
+  margin-top: 1rem;
+  width: 100%;
+}
+
+@media (min-width: 561px) {
+  .search-form {
+    margin-top: 2.5rem;
+  }
+
+  .search-form__input {
+    max-width: 300px;
+  }
+
+  .search-form__button {
+    margin-top: 0;
+    margin-left: 1rem;
+    width: unset;
```

Comment:
If at all possible, its better to assign a property to a value rather than remove a previously assigned one like this.
```
    width: auto;
```
Personally I think this would be a good case for using flexbox on the container and using that to handle the alignment of the various elements, this would also make it simpler to change things like the alignment with `flex-direction`